src/screens/Departures/components/DateNavigator.tsx

```diff
@@ -0,0 +1,145 @@
+import {StyleSheet, useTheme} from '@atb/theme';
+import React, {Dispatch, SetStateAction} from 'react';
+import {View} from 'react-native';
+import Button from '@atb/components/button';
+import {Language, useTranslation} from '@atb/translations';
+import {
+  formatToClock,
+  formatToShortDate,
+  formatToSimpleDateTime,
+  isInThePast,
+} from '@atb/utils/date';
+import {ArrowLeft, ArrowRight} from '@atb/assets/svg/icons/navigation';
+import {Date as DateIcon} from '@atb/assets/svg/icons/time';
+import {SearchTime} from '../Departures';
+import {addDays, isSameDay, isToday, parseISO} from 'date-fns';
+import DepartureTimeSheet from '../../Nearby/DepartureTimeSheet';
+import {useBottomSheet} from '@atb/components/bottom-sheet';
+import DeparturesTexts from '@atb/translations/screens/Departures';
+
+type DateNavigationProps = {
+  searchTime: SearchTime;
+  setSearchTime: Dispatch<SetStateAction<SearchTime>>;
+};
+
+export default function DateNavigation({
+  searchTime,
+  setSearchTime,
+}: DateNavigationProps): JSX.Element {
+  const styles = useStyles();
+  const {theme} = useTheme();
+  const {t, language} = useTranslation();
+  const canNavigateToPreviousDay = isToday(parseISO(searchTime.date));
+
+  const searchTimeText =
+    searchTime.option === 'now'
+      ? t(DeparturesTexts.dateNavigation.today)
+      : formatToTwoLineDateTime(searchTime.date, language);
+  const a11ySearchTimeText =
+    searchTime.option === 'now'
+      ? t(DeparturesTexts.dateNavigation.today)
+      : formatToSimpleDateTime(searchTime.date, language);
+
+  const onSetSearchTime = (time: SearchTime) => {
+    if (isInThePast(time.date)) {
+      setSearchTime({
+        date: new Date().toISOString(),
+        option: 'now',
+      });
+      return;
+    }
+    setSearchTime(time);
+  };
+
+  const {open: openBottomSheet} = useBottomSheet();
+  const onLaterTimePress = () => {
+    openBottomSheet((close, focusRef) => (
+      <DepartureTimeSheet
+        ref={focusRef}
+        close={close}
+        initialTime={searchTime}
+        setSearchTime={onSetSearchTime}
+      ></DepartureTimeSheet>
+    ));
+  };
+
+  return (
+    <View style={styles.dateNavigator}>
+      <Button
+        onPress={() => {
+          setSearchTime(changeDay(searchTime, -1));
+        }}
+        text={t(DeparturesTexts.dateNavigation.prevDay)}
+        type="inline"
+        mode="tertiary"
+        icon={ArrowLeft}
+        disabled={canNavigateToPreviousDay}
```

Comment:
Er det noe feil med navngivingen her? Knappen for å gå tilbake til forrige dag er `disabled` hvis `canNavigateToPreviousDay`. Ser du ikke har endret navngivingen her, men uansett lurt å fikse opp i hvis det er feil.

Reply:
Ja sant! Oppdatert til `disablePreviousDayNavigation`